vehicle/raw_vehicle_cmd_converter/CMakeLists.txt

```diff
@@ -1,57 +1,35 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Out of curiosity: I chose the same version number in my PR but is there really a specific cmake version we require? For example why not 3.10?

Reply:
I don't really have particular reason.
It's just that packages in Autoware.Auto require 3.5, and I made it consistent with them.

Reply:
ok, will do the same in my ports